### PR TITLE
Resources Page + Carousel Loop

### PR DESCRIPTION
**Pull request by Bi1ku**

### CODE REVIEW

File: src/pages/Resources/data.js [hunk 1]
```diff
@@ -0,0 +1,310 @@
+export const resourcesData = [
```
Comment (Bi1ku) [inline src/pages/Resources/data.js:1]: please help, i'm slowly losing it.

File: src/pages/Resources/data.js [hunk 2]
```diff
@@ -0,0 +1,310 @@
...
+        ],
+      },
+      {
+        title: 'Project SHIFT',
+        resources: [
+          { title: 'P.S. 201Q', link: 'https://www.ps201.org/' },
+          { title: 'P.S. 219Q', link: 'https://www.psms219.org/' },
+          { title: 'M.S. 216Q', link: 'https://www.ryan216q.org/' },
+        ],
+      },
```
Comment (the-eyesack) [inline src/pages/Resources/data.js]: no need for Project SHIFT schools

File: src/pages/Resources/data.js [hunk 3]
```diff
@@ -0,0 +1,310 @@
...
+          {
+            title: 'Lorem Ipsum',
+            link: '#',
+          },
+        ],
+      },
+    ],
+  },
+  {
+    title: 'Media',
```
Comment (the-eyesack) [inline src/pages/Resources/data.js]: No need to have this- if anything we can have an API to display our videos

File: src/pages/Resources/data.js [hunk 4]
```diff
@@ -0,0 +1,310 @@
...
+          {
+            title: 'FIRST Impact Video 2015',
+            link: 'https://www.youtube.com/watch?v=-2ma0OHa9PY',
+          },
+        ],
+      },
+    ],
+  },
+  {
+    title: 'FIRST Awards',
```
Comment (the-eyesack) [inline src/pages/Resources/data.js]: no need

File: src/pages/Resources/data.js (hunk 1 above)
Comment (the-eyesack) [reply src/pages/Resources/data.js:1]: OWEN YOU CANT LOSE IT YET

File: src/pages/Resources/data.js [hunk 5]
```diff
@@ -0,0 +1,310 @@
...
+          {
+            title: 'HawkPod Playlist',
+            link: 'https://www.youtube.com/watch?v=Y7tbsk4PNFc&list=PL1CVNaoFuZz2_9e5Eq_-7MqhVP2LiuBdZ',
+          },
+        ],
+      },
+    ],
+  },
+  {
+    title: 'Purus it amet',
```
Comment (the-eyesack) [inline src/pages/Resources/data.js]: remove pls ty

File: src/pages/Resources/index.jsx [hunk 6]
```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { resourcesData } from './data';
+
+function ResourcesPage() {
+  return (
+    <div className='grid lg:grid-cols-3 sm:grid-cols-2 grid-cols-1 p-10 gap-y-10 gap-x-4'>
```
Comment (the-eyesack) [inline src/pages/Resources/index.jsx]: add a 

`<h1 className="pageTitle">Resources</h1>`

**Review by the-eyesack — CHANGES_REQUESTED** [review]
good job!!!!!!!!!!!!! just fix those few things and we good to go

**Pull request merged**